agents/PAW-04 Documenter.agent.md

```diff
@@ -510,45 +510,20 @@ Before pushing:
 
 ## Hand-off
 
-### After Initial Documentation
+{{HANDOFF_INSTRUCTIONS}}
 
-```
-Documentation Complete - Docs PR Opened
-
-I've created comprehensive documentation and opened the Docs PR (add actual number when known) at:
-`<target_branch>_docs` → `<target_branch>`
-
-The PR includes:
-- **Docs.md** - Detailed documentation serving as the authoritative technical reference at .paw/work/<feature-slug>/Docs.md
-- [List of updated project documentation files, which are based on/derived from Docs.md]
-
-Key documentation highlights:
-- Complete architecture and design decisions
-- Comprehensive usage guide with examples (when applicable)
-- Full technical reference and API documentation
-- Edge cases, limitations, and testing approach documented
-
-Please review Docs.md for completeness and technical accuracy. It's designed to be the go-to reference for engineers working with this implementation.
-
-Next: After Docs PR is merged, invoke PR Agent (Stage 05) to create the final PR to main.
-```
-
-### After Addressing Review Comments
-
-```
-Review Comments Addressed - Changes Pushed and Summary Posted
-
-I've addressed all review comments on the Docs PR with focused documentation improvements.
+### Documentation Handoff
 
-Changes made:
-- [List of commits with brief descriptions]
+**prs strategy** (Docs PR opened):
+- All Modes: Pause after Docs PR (wait for human review)
+- After PR review comments: Say `review` to address comments (handoff to self with review context)
```

Comment:
'review' might not be the right word here. Perhaps "address comments". Or "check pr".